docs/usage.md

```diff
@@ -348,83 +349,136 @@ This will affect the add button text and index/form page titles.
 
 ---
 
-# Navigation Items
+# Navigation
 
-When you use the generators, a link to the section appears in the main navigation of the admin. This is done by automatically adding to `app/controllers/concerns/fae/nav_items.rb`. However, this file is available for you to customize the nav however you'd like.
+When using Fae's installer and generators, navigation items to the objects you create will automatically get added to the side nav of the admin. Behind the scenes the generator is updating `app/models/concerns/fae/navigation_concern.rb`. However, this file is available for you to customize the nav however you like.
 
-The navigation is built of of the array set in the `nav_items` method. Each array item is a hash with these available keys:
+## Updating navigation_concern.rb
 
-| Key | Type | Description |
-| --- | ---- | ----------- |
-| text | string | the link's text |
-| path | string or named route | the link's href (defaults to '#') |
-| class | string | an added class to the link |
-| sublinks | array of hashes | nested links to be displayed in a dropdown |
-
-## Named Routes in nav_items.rb
-
-Since the `nav_items` concern hooks directly into Fae, named routes need context using the following prefixes:
+To define your own navigation structure you just need to update the `structure` method within `app/models/concerns/fae/navigation_concern.rb`. Here's a look at an example:
 
 ```ruby
-def nav_items
+def structure
   [
-    # use `main_app.` for application routes (even in your admin)
-    { text: 'Cities', path: main_app.admin_cities_path },
-    # use `fae.` for Fae routes
-    { text: 'Pages', path: fae.pages_path }
+    item('Products', subitems: [
+      item('Apparel', class: 'custom-class', path: admin_apparel_path),
+      item('Beverage', subitems: [
+        item('Wines', subitems: [
+          item('Releases', path: admin_releases_path)
+        ]),
+        item('Board Games', path: admin_board_games_path)
+      ]),
+    ]),
+    item('Pages', subitems: [
+      item('Home', path: fae.edit_content_block_path('home')),
+      item('About Us', path: fae.edit_content_block_path('about_us'))
+    ])
   ]
 end
 ```
 
-## Sublinks
+`structure` is an array of nav items which are defined with the `item` method. The `item` method takes a title as the first argument, followed by a hash options. To nest subitems within an item, pass an array of items into the `subitems` option.
```

Comment:
> Followed by a hash options

Do you mean "followed by a hash" or "followed by a hash of options" or "and a second argument as a hash of options" ?